<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncluirLivro</class>
 <widget class="QDialog" name="IncluirLivro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Incluir Livro</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>300</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>INCLUIR NOVO LIVRO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nomeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="precoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Pre√ßo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="autorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Autor:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomeLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="precoLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="autorLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IncluirLivro</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IncluirLivro</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
